plans/migrate.pp

```diff
@@ -14,6 +14,7 @@
   Peadm::SingleTargetSpec $new_primary_host,
   Optional[String] $upgrade_version = undef,
   Optional[Peadm::SingleTargetSpec] $replica_host = undef,
+  Optional[Peadm::SingleTargetSpec] $primary_postgresql_host = undef,
 ) {
   # pre-migration checks
   out::message('This plan is a work in progress and it is not recommended to be used until it is fully implemented and supported')
```

Comment:
Below, we should add the $primary_postgresql_host node into the $all_hosts collection like we do for $replica_host